CLOUD-EDGE-CLIENT_COORDINATION_CG_Proposal

```diff
@@ -0,0 +1,11 @@
+1、Group name
+CLOUD-EDGE-CLIENT COORDINATION COMMUNITY GROUP
+
+2、Group Description
+The mission of this group is to provide mechanisms and interfaces between Central Cloud, Edge-Cloud, and Client for computing workload offloading and orchestration. 
+The typical use cases include AI acceleration, cloud gaming, streaming acceleration, etc. The goal is to design a set of new APIs and mechanisms 
+that enable computing workload offloading and orchestration between Central Cloud, Edge Cloud, and Client. It should leverage existing mechanisms as much as possible 
```

Comment:
One thing I've been thinking about generally is that there are two kinds of "edge" computer, those managed as an extension of the cloud (Cloud-Edge), and those that are actually clients, e.g. other client computers on the LAN.   Some of the use cases I have been thinking about are for offloading to "Distributed Compute Resources" which I have been using to avoid getting into arguments about the definition of "Edge".   However - the term "Edge-Cloud" may be too restrictive here.   How about we simply use "Cloud, Edge, and Client" for the charter?   We can refine the definitions later if we want (e.g. add terms for Central Cloud, Edge-Cloud, Edge-Client, Client, etc.).